includes/templates/single-web-story.php

```diff
@@ -26,7 +28,12 @@
 
 the_post();
 
-?>
-<!DOCTYPE html>
-<?php
-the_content();
+$post = get_post();
+
+if ( $post instanceof WP_Post ) {
+	// TODO: Fix issue with AMP plugin.
+	ob_end_clean();
```

Comment:
Removing this breaks things.

Need to ensure the AMP plugins output buffer runs after ours.